packages/zowe-explorer/src/trees/job/JobTableView.ts

```diff
@@ -0,0 +1,246 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { IZoweJobTreeNode, Table, TableBuilder, TableViewProvider } from "@zowe/zowe-explorer-api";
+import { SharedTreeProviders } from "../shared/SharedTreeProviders";
+import { ExtensionContext } from "vscode";
+import { JobActions } from "./JobActions";
+import { ZoweJobNode } from "./ZoweJobNode";
+import { SharedUtils } from "../shared/SharedUtils";
+import { SharedContext } from "../shared/SharedContext";
+import { IJob } from "@zowe/zos-jobs-for-zowe-sdk";
+
+export class JobTableView {
+    private static cachedChildren: IZoweJobTreeNode[];
+    private static contextOptions: Record<string, Table.ContextMenuOpts> = {
+        getJcl: {
+            title: "Get JCL",
+            command: "get-jcl",
+            callback: {
+                fn: JobTableView.getJcl,
+                typ: "single-row",
+            },
+        },
+        displayInTree: {
+            title: "Display in tree",
```

Comment:
Should this menu item title use title case?
```suggestion
            title: "Display in Tree",
```